SmartDeviceLinkTests/DevAPISpecs/SDLVoiceCommandManagerSpec.m

```diff
@@ -155,6 +158,50 @@ @interface SDLVoiceCommandManager()
                 });
             });
         });
+
+        context(@"if it has voice commands to upload with one voice command strings contains an empty string", ^{
+            beforeEach(^{
+                testManager.voiceCommands = @[testVoiceCommand3, testVoiceCommand4, testVoiceCommand5];
+            });
+
+            // should queue another operation
+            it(@"should queue another operation", ^{
+                expect(testManager.transactionQueue.operations).to(haveCount(2));
+                expect(testManager.voiceCommands).to(haveCount(1));
+                expect(testManager.voiceCommands.firstObject.voiceCommands).to(haveCount(2));
```

Comment:
Can we add tests here to make sure that `@[@"Test 3", @" ", @"Test 4"]` becomes `@[@"Test 3", @"Test 4"]`